### PR TITLE
Make columns not-nullable by default

### PR DESCRIPTION
**Pull request by danpalmer**

### CODE REVIEW

File: routemaster/db/model.py [hunk 1]
```diff
@@ -50,8 +53,8 @@
     Column('forced', Boolean, default=False),
 
     # Null indicates starting a state machine
-    Column('old_state', String, nullable=True),
-    Column('new_state', String),
+    NullableColumn('old_state', String),
+    NullableColumn('new_state', String),
```
Comment (PeterJCLaw) [inline routemaster/db/model.py]: Is this change intentional? (this column was previously non-nullable)

Comment (danpalmer) [reply routemaster/db/model.py]: Actually columns are nullable by default, and we even use the null value here when we delete a label.

Comment (PeterJCLaw) [reply routemaster/db/model.py]: So they are. My question was more around the underling intention. As discussed IRL though we do depend on this being nullable; so good to have this clarified in the models.
